pepper-apis/studybuilder-cli/studies/lms/study-events.conf

```diff
@@ -1130,7 +1130,7 @@
             "isDynamic": false
           }
         ],
-        "pdfAttachments": []
+        "pdfAttachments": [ { "pdfName": "lmsproject-release", "generateIfMissing": true } ]
```

Comment:
As a point for cross training, do you have to deploy the study again when you change this?